trade-finance-manager-api/src/v1/emails/MIA-confirmation/bss-email-variables.js

```diff
@@ -0,0 +1,23 @@
+const { generateFacilitiesListString } = require('../../helpers/notify-template-formatters');
+const getFacilitiesByType = require('../../helpers/get-facilities-by-type');
+
+const bssEmailVariables = (deal) => {
+  const { facilities } = deal;
+
+  const { bonds, loans } = getFacilitiesByType(facilities);
+  const bssList = generateFacilitiesListString(bonds);
+  const ewcsList = generateFacilitiesListString(loans);
+
+  const facilityLists = { bssList, ewcsList };
+
+  const emailVariables = {
+    bssList: facilityLists.bssList,
+    showBssHeader: facilityLists.bssList ? 'yes' : 'no',
```

Comment:
is it supposed to be `yes` / `no`? or can we use `true` / `false`?

Reply:
I think it does unfortunately, for Notify template. Other email variable functions use this :/ . Would rather be consistent. Could double check later?